### DISCUSSION TITLE
Is it possible to work without coords?

### DISCUSSION THREAD

**Discussion by mikauser001** [themustafaomar/jsvectormap#134] [Q&A]
Hi,

I am a big fan of  your work, Is it possible to integrate
something like hard coded coordinates for each country in the future? 

Example

{ name: 'Egypt', coords: [26.8206, 30.8025] }, could become names: ['Egypt', ....] where each country has default coord(s), that can be overwritten at any time through coords: [x, y]

**Comment by themustafaomar** [comment]
Hi @matidfk thanks for the kind works!

Actually I'm not planning to add this in the future, even if I decided to make it I would create an external array of objects each object has a unique country name and its coordinates, then you can find your country and get its coordinates and pass it to  marker config, this makes jsvectormap lightweight.
